pkg/auth/rbac.go

```diff
@@ -67,10 +68,11 @@ type rbacConnection struct {
 	dstPort     uint32
 }
 
-func NewRbac(workloadObj *bpf.BpfKmeshWorkload) *Rbac {
+func NewRbac(workloadObj *bpf.BpfKmeshWorkload, workloadCache cache.WorkloadCache) *Rbac {
 	return &Rbac{
-		policyStore: newPolicystore(),
-		bpfWorkload: workloadObj,
+		policyStore:   newPolicystore(),
+		workloadCache: workloadCache,
+		bpfWorkload:   workloadObj,
```

Comment:
Why add workload info to the rbac cache？

Reply:
because this patch is also removing the global cache var